build.gradle

```diff
@@ -16,18 +16,20 @@
 
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
-buildscript {
+buildscript { 
+    ext.kotlin_version = '1.3.50'
+
     repositories {
         google()
         jcenter()
-        
+
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:3.4.2'
-        
+
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
-    }
+    classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version" }
```

Comment:
I'm assuming there's not way to add the Kotlin stuff only to the playbilling module (or maybe we want to use Kotlin in other places as well?)

Reply:
After I've got the library in some rough shape I'm going to bundle it, include it in Chrome and see what the binary size impact is. If it's not too big, then I'm happy spreading Kotlin to other parts of ABH, otherwise I'll probably have to rewrite this all in Java and we can get rid of Kotlin.

OTOH, maybe we could just spread Kotlin everywhere and only revert the play billing part if needed.

Reply:
SGTM